src/contracts/gov/ODGovernor.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.19;
+
+import {IVotes} from '@openzeppelin/governance/utils/IVotes.sol';
+import {IERC165} from '@openzeppelin/utils/introspection/IERC165.sol';
+import {IGovernor} from '@openzeppelin/governance/IGovernor.sol';
+
+import {TimelockController} from '@openzeppelin/governance/TimelockController.sol';
+
+import {Governor} from '@openzeppelin/governance/Governor.sol';
+import {GovernorSettings} from '@openzeppelin/governance/extensions/GovernorSettings.sol';
+import {GovernorCompatibilityBravo} from '@openzeppelin/governance/compatibility/GovernorCompatibilityBravo.sol';
+import {GovernorVotes} from '@openzeppelin/governance/extensions/GovernorVotes.sol';
+import {GovernorVotesQuorumFraction} from '@openzeppelin/governance/extensions/GovernorVotesQuorumFraction.sol';
+import {GovernorTimelockControl} from '@openzeppelin/governance/extensions/GovernorTimelockControl.sol';
+
+contract ODGovernor is
+  Governor,
+  GovernorSettings,
+  GovernorCompatibilityBravo,
+  GovernorVotes,
+  GovernorVotesQuorumFraction,
+  GovernorTimelockControl
+{
+  /**
+   * @dev GovernorSettings unit of time:
+   * 1 = 12 second, 5 = 1 minute, 300 = 1 hour, 7200 = 1 day, 50400 = 1 week
+   *
+   * @param _token is protocolToken (Open Dollar Governance, ODG)
+   * @param _timelock grace period to allow rage quit between proposal success and execution
+   */
+  constructor(
+    address _token,
+    TimelockController _timelock
+  )
+    Governor('ODGovernor')
+    GovernorSettings(1, 15, 0)
```

Comment:
this should be set by variables in the deployment scripts, but its good for now.